### PR TITLE
first try

### PR DESCRIPTION
**Pull request by VihanRajpara**
hii

### CODE REVIEW

**Comment by VihanRajpara** [comment]
done

**Comment by VihanRajpara** [comment]
gg

**Review by VihanRajpara — COMMENTED** [review]
done

**Review by VihanRajpara — COMMENTED** [review]
done

File: src/App.js [hunk 1]
```diff
@@ -5,8 +5,8 @@ import { ColorModeContext, useMode } from "./theme";
 import ForgotPassword from "./pages/Login/ForgotPassword";
 import NotFoundPage from "./pages/NotFoundPage";
 import Useraccess from "./pages/Useraccess";
-import CROforInvestment from "./pages/CROforInvestment";
-
+import CROTab from "./pages/CROTab.jsx";
+import "./App.css";
 function App() {
   const [theme, colorMode] = useMode();
```
Comment (VihanRajpara) [inline src/App.js:12]: done

File: src/App.css [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  .header{
+    background-color: #18446B;
+    font-size: 1.1em;
+    font-weight: bolder;
+    color: #eeeeee;
+    text-align: center;
+    padding: 0.3em;
+    text-transform: uppercase;
+    font-family: Verdana, Arial, sans-serif;
+    border-radius: 0.25rem;
```
Comment (ajay260902) [inline src/App.css:14]: good

**Review by ajay260902 — APPROVED** [review]

**Pull request merged**